mlos_bench/mlos_bench/services/remote/azure/azure_services.py

```diff
@@ -445,11 +445,20 @@ def _check_deployment(self, params: dict) -> Tuple[Status, dict]:
             deployment_name=config["deploymentName"],
         )
 
-        response = requests.head(url, headers=self._get_headers(), timeout=self._request_timeout)
+        response = requests.get(url, headers=self._get_headers(), timeout=self._request_timeout)
         _LOG.debug("Response: %s", response)
 
-        if response.status_code == 204:
```

Comment:
Any other tests or handling required for non-200 responses?
Is it possible to get a 204 again?